README.md

```diff
@@ -0,0 +1,29 @@
+# High Level Controller
+
+## Overview
+
+The control-high repository consists of the High-Level Controller and the MHIST submodule of the SOZO Robotics Project.
+
+The High-Level Controller (HLC) receives the area to cover from the Swarm Algorithm (tbd, currently mocked), gets updates about the local map and the current position in the map from [SLAM](https://github.com/codeuniversity/slam) and feedback about the (sucessful) leg movement from the [gait controller](https://github.com/codeuniversity/control-gait).
+The Planning Alogorithm creates the high level actions (hla) to ensure a sucessful exploration of the area the robot should cover. You can test the algorithms with the pytests defined in /test. These hla ("move forward" or "turn by 90 degrees") are forwarded to the gait controller.
+
+## Project Structure
+
+![hlc_sequence_diagram](./images/hlc_sequence_diagram.png)
+
+The "Monitor" is the main Process, that is starting the subprocesses "Queues", "Receiver", "Supplier" and "Plan". The receiver subscribes to [MHIST](https://github.com/alexmorten/mhist/tree/2b4bed690fb6b38bf6e0c13f7b4f67c05c5d9c52), a simple on disc measurement data base that stores and redistributes measurements consisting of a name, a value and optionally a timestamp through grpc.
```

Comment:
`disc` -> `disk`
the `timestamp` is only optional when sending a measurement; it will always be present when getting measurements from mhist (when not sent, mhist uses the current time :) )